exercise_1.py

```diff
@@ -51,20 +51,23 @@
 
 # Run the QUBO on the solver from your config file
```

Comment:
I wonder if the 'from your config file' part of the comment is misleading since the IDE doesn't use config files in the same way as a local setup and we don't need anything special in a config file to use the `topology__type` parameter. Instead I might say something like
_Run the QUBO on a solver with the specified topology_